easybuild/easyconfigs/f/FASTA/FASTA-36.3.8i-foss-2022b.eb

```
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = "FASTA"
+version = "36.3.8i"
+
+homepage = 'http://fasta.bioch.virginia.edu'
+description = """The FASTA programs find regions of local or global (new) similarity between 
+protein or DNA sequences, either by searching Protein or DNA databases, or by identifying 
+local duplications within a sequence."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
```

we can keep this at `GCC` level instead of `foss`, since it doesn't have any dependency on MPI, BLAS/LAPACK or FFTW
```suggestion
toolchain = {'name': 'GCC', 'version': '12.2.0'}
```